### PR TITLE
Add a note about `@external_resource` not being there yet

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: lib/elixir/lib/module.ex [hunk 1]
```diff
@@ -252,6 +252,9 @@ defmodule Module do
   in case any of the external resources change, see for example:
   [`mix compile.elixir`](https://hexdocs.pm/mix/Mix.Tasks.Compile.Elixir.html).
 
+  Even if the given external file does not exist, the module still
+  adds a dependency on it.
```
Comment (wojtekmach) [inline lib/elixir/lib/module.ex:256]: The opposite is not currently true, if we remove the file, the module is not recompiled. Is it worth changing?

Comment (josevalim) [reply lib/elixir/lib/module.ex:256]: Maybe we should fix it.

Comment (josevalim) [inline lib/elixir/lib/module.ex:256]:
```suggestion
  If the external resource does not exist, the module still adds a
  dependency on it, causing it to be recompiled as soon as the
  file is added.
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**